react/ProductHighlights.test.tsx

```diff
@@ -54,9 +54,7 @@ test("render all teasers' highlights", () => {
 
 test('render nothing if wrong type is passed', () => {
   const { container } = render(
-    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-    // @ts-ignore
-    <ProductHighlights type="">
+    <ProductHighlights>
```

Comment:
@victorhmp

Because of Lint's mistakes. In the old version it caused a Lint error

Reply:
Can you revert the changes in this file and see if the lint bot will complain here? It shouldn't do it. And if it does, it shouldn't affect your PR since you were not changing the tests initially.

Reply:
@icazevedo 

Corrections made.